backend/browser.py

```diff
@@ -41,7 +41,18 @@ def _unzip_to_plugin_dir(self, zip, name, hash):
             return False
         zip_file = ZipFile(zip)
         zip_file.extractall(self.plugin_path)
-        Popen(["chown", "-R", "deck:deck", self.plugin_path])
+        USER = None
+        cmd = "who | awk '{print $1}' | sort | head -1"
+        while USER == None:
+            USER_LIST = subprocess.Popen(cmd, stdin=subprocess.PIPE, stdout = subprocess.PIPE, stderr = subprocess.PIPE, shell=True)
+            for get_first in USER_LIST.stdout:
+                name = get_first.decode().strip()
+                if name is not None:
+                    USER = name
+                    break
+            time.sleep(1)
```

Comment:
As already discussed this call to time.sleep(1) seems to break based upon how imported the time module. More testing can be done once this is resolved.

Reply:
Changed this and identified a few other import issues as well, specifically i was using the subprocess prefix and I missed the PIP import. Also used the asyncio sleep in main.py instead of time wince that is an async function which I missed the first push. Everything should be testable now.